### PR TITLE
Fix support of raw files

### PR DESCRIPTION
**Pull request by tmp-hallenser**

### CODE REVIEW

**Comment by ildyria** [comment]
Code wise, no complaints.

I will test it tomorrow with uploading a pdf and some text files ! :D

**Comment by tmp-hallenser** [comment]
Unfortunately, there some more work to do. So far, I only fixed the extraction of metadata. 

Open is still the creation of jpgs.

**Review by ildyria — APPROVED** [review]

**Comment by ildyria** [comment]
At least the current code does not break when I upload ... e.g. pdf. :+1:

**Comment by tmp-hallenser** [comment]
Added the missing lines to create medium, small and thumb jpg files in case of raw files.

Front-end needs to be updated to show work correctly.

Currently, it requires exiftool to work since we need the image dimensions.

**Comment by tmp-hallenser** [comment]
Just created a PR for php-exif with imagick as an additional adapter. This will allow uploading raw files without exiftool being installed (some minor changes needed).

**Comment by ildyria** [comment]
Code wise, no complaints.
Still need to test. :smiley:

**Review by ildyria — APPROVED** [review]

**Comment by ildyria** [comment]
Works fine. I don't have actual raw image to test with on this laptop.

**Comment by ildyria** [comment]
@tmp-hallenser merge when you feel like it is ready. :+1:

**Comment by tmp-hallenser** [comment]
Now it should be ready for testing

File: setup_exiftool.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 set -e
 
 wget https://exiftool.org/Image-ExifTool-11.99.tar.gz
-tar -zxvf Image-ExifTool-11.77.tar.gz
+tar -zxvf Image-ExifTool-11.99.tar.gz
```
Comment (ildyria) [inline setup_exiftool.sh:5]: fixed in master :)

**Review by ildyria — APPROVED** [review]

**Comment by tmp-hallenser** [comment]
The PR is now ready to merge. Any objections? @ildyria

**Pull request merged**